doc/nrf/libraries/others/emds.rst

```diff
@@ -128,9 +130,9 @@ The :c:func:`emds_store_time_get` function estimates the required worst-case tim
    t_\text{store} = t_\text{base} + \sum_{i = 1}^n \left(t_\text{entry} + t_\text{word}\left(\left\lceil\frac{s_\text{ate}}{s_\text{block}}\right\rceil + \left\lceil\frac{s_i}{s_\text{block}}\right\rceil \right)\right)
 
 where :math:`t_\text{base}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :math:`t_\text{entry}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :math:`t_\text{word}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`.
-:math:`s_i` is the size of the :math:`i`\ th entry in bytes and :math:`s_\text{block}` is the number of bytes in one word of flash.
+:math:`s_i` is the size of the :math:`i`\ th entry in bytes and :math:`s_\text{block}` is the number of bytes in one word of flash/RRAM.
 These can be found by looking at datasheets, driver documentation, and the configuration of the application.
-:math:`s_\text{ate}` is the size of the allocation table entry used by the EMDS flash module, which is 8 B.
+:math:`s_\text{ate}` is the size of the allocation table entry used by the EMDS flash/RRAM module, which is 8 B.
```

Comment:
```suggestion
:math:`s_\text{ate}` is the size of the allocation table entry used by the EMDS flash/RRAM module, which is 8 B if NVS is used, else it is 16 B for ZMS.
```

Reply:
The ate size is 8 regardless of ZMS or NVS. It is defined in emds_flash.c.